DxDispatch/src/dxdispatch/DmlDispatchable.h

```diff
@@ -1,13 +1,31 @@
 #pragma once
+#include "DirectMLHelpers\DmlSerializedGraphDesc.h"
+
 
 class DmlDispatchable : public Dispatchable
 {
 public:
+    struct BindPoint
```

Comment:
We can move out the `BindPoint` and `BindPoints` struct definition from the DmlDispatchable in Model.h file and remove it from here. Right now the same struct definition is in 2 places.

Reply:
done